### ISSUE TITLE
Soundfonts are still broken

### ISSUE THREAD

**Issue by Focus78** [joncampbell123/dosbox-x#2811]
This this being fixed?

**Comment by emendelson** [comment]
Duplicates #2774 and #2801.

**Comment by Focus78** [comment]
@emendelson

This this being fixed?

**Comment by emendelson** [comment]
I have no way of knowing what the developers' priorities are.

**Comment by rebb** [comment]
Both of these are from the same author it seems, and from this month. Let's just say that i think the devs are doing great work here, and compared to other repositories i've seen are very responsive to problems. Give them a break, or at least some time to get around to this :)

**Comment by emendelson** [comment]
@rabb Agreed. The devs are doing an absolutely great job, and they deserve an enormous amount of gratitude for the unpaid, often unappreciated, work that they're doing.

**Comment by Dobby233Liu** [comment]
maybe nobody will look into this (i don't know anything too), but still, please fill this issue template, i'm sure it can help people investigate this if anybody does: https://github.com/joncampbell123/dosbox-x/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

**Comment by Focus78** [comment]
@Dobby233Liu 

https://github.com/joncampbell123/dosbox-x/issues/2774

Windows x86 platform.

**Comment by Dobby233Liu** [comment]
i know, i might not made it clear but please fill the template completely here, though if you can't then nevermind, i'm not demanding you

**Comment by Focus78** [comment]
It's not hard to replicate.

Make sure sound canvas set and the path indicated in the config file.

Start the SDL2 x86 build and boot any game with a sound option for sound canvas.

The music will be all screwed and distorted.

An example off the top of my head that I tried last was the WeedsGM3.sf2 soundfont and Duke Nukem 3D.

**Comment by Focus78** [comment]
@Dobby233Liu

What was the point in asking me for instructions to reproduce if you weren't going to do anything?

**Comment by joncampbell123** [comment]
I'm unable to work on DOSBox-X because my professional work is taking priority. Is it all 32-bit SDL2 builds, or just Windows?

**Comment by Focus78** [comment]
@joncampbell123

The x86 SDL2 build on windows as of the current release.

The SDL1 build does not experience the same problem.

**Comment by joncampbell123** [comment]
MinGW or VS2019 build?

**Comment by Focus78** [comment]
@joncampbell123

VS
